jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/ProcessInstanceInfo.java

```diff
@@ -103,6 +104,16 @@ public ProcessInstanceInfo(ProcessInstance processInstance) {
         this.processInstance = processInstance;
         this.processId = processInstance.getProcessId();
         startDate = new Date();
+
+        // If we are creating a second Process Instance Info for the same process instance,
+        // it should not generate a new start date
+        if (this.processInstance != null) {
```

Comment:
You can remove this check, as it will always be true (notice that a null pointer exception will be thrown 2 lines above if processInstance were null)